src/app/commands/bounty/Bounty.ts

```diff
@@ -182,9 +183,9 @@ export default class Bounty extends SlashCommand {
                             required: true,
                         },
                         {
-                            name: 'tag',
+                            name: 'keywords',
```

Comment:
Minor, but Re: the name keywords vs tag, I think  `tag` as a name is ubiquitous language for software applications, and has usability connotations associated (I.e. I can search in aggregate for this tag later on, and GUI views can be grouped by tag, etc..).

Keywords also is ubiquitous language, but in the context of text search and indexing, rather than GUIs.

Reply:
Renamed it. It was inconsistent with the modal create anyway.